### PR TITLE
Feature/aws profile

### PR DESCRIPTION
**Pull request by gsoyka**
Added the ability to configure an AWS Profile; however switching profiles currently requires restarting the application.

### CODE REVIEW

File: app/components/Home.js [hunk 1]
```diff
@@ -60,6 +60,14 @@ class Home extends Component {
       }
     );
 
+    const profile = localStore.get(availableSettings.profile);
```
Comment (smblee) [inline app/components/Home.js]: No need for you to do this. This is only needed if the homepage needs this value from the state

File: app/menu.js [hunk 2]
```diff
@@ -45,17 +45,17 @@ export default class MenuBuilder {
 
   buildDarwinTemplate() {
     const subMenuAbout = {
-      label: 'Electron',
+      label: 'Parameter Store Manager',
```
Comment (smblee) [inline app/menu.js]: Good change!

File: app/menu.js [hunk 3]
```diff
@@ -66,6 +66,14 @@ export default class MenuBuilder {
         },
         { label: 'Show All', selector: 'unhideAllApplications:' },
         { type: 'separator' },
+        {
+          label: 'Restart',
+          accelerator: 'Command+Shift+R',
+          click: () => {
+            app.relaunch();
```
Comment (smblee) [inline app/menu.js]: This seems a little odd. Relaunch => quit ? Is this a convention in electron?

**Comment by smblee** [comment]
Thank you for the pull request! I went ahead and resolved the lint errors (as they were small); there is one outstanding question but it seems to work fine. Also good catch on the About menu description.

**Comment by smblee** [comment]
Also added another issue (#77) to hopefully eliminate the need to restart at all.

**Comment by gsoyka** [comment]
Thanks for the tip about `yarn lint`; I'm a novice at best with nodejs, so thanks for cleaning that up for me.

**Pull request merged**